lua/trouble/init.lua

```diff
@@ -110,12 +110,7 @@ function Trouble.help()
     height = height + 1
   end
   -- help
-  vim.lsp.util.open_floating_preview(lines, "markdown", {
-    border = "single",
-    height = 20,
-    offset_y = -2,
-    offset_x = 2,
-  })
+  vim.lsp.util.open_floating_preview(lines, "markdown", { border = "single" })
```

Comment:
I removed the other window settings here because the preview window doesn't have them and I was using that as a reference, and it seems like the floating preview will automatically figure out the dimensions.